DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: etn
 Title: Access Data from the European Tracking Network
-Version: 2.3.0
+Version: 2.3.1.9000
```

Comment:
Don't update the version number until release. It should stay `<latest-release>.9000` (so `2.2.1.9000`). See https://github.com/inbo/etn/issues/334